packages/flame/lib/src/components/base_component.dart

```diff
@@ -20,14 +18,6 @@ import 'mixins/has_game_ref.dart';
 abstract class BaseComponent extends Component {
```

Comment:
Now that the children were moved into `Component`, this description needs to be updated.

Reply:
Yeah, I'm thinking of refactoring away `BaseComponent` completely since it is mostly the effects system left, and that can also be a component. Lots of docs and descriptions and re-ordering left to to be done. :)